draggablepanel/src/main/java/com/github/pedrovgs/DraggableView.java

```diff
@@ -366,7 +373,9 @@ public boolean isClosed() {
     if (activePointerId == INVALID_POINTER) {
       return false;
     }
-    viewDragHelper.processTouchEvent(ev);
+    if(getContext().getResources().getConfiguration().orientation == Configuration.ORIENTATION_PORTRAIT){
+      viewDragHelper.processTouchEvent(ev);
+    }
```

Comment:
In case of usage a video player such as ExoPlayer, this processes the drag event when you try to hover the player controls, breaks the user experience about controlling the player. A control has been added for that purpose. Could be optional for people who doesn't have hover for player controls and aims to drag even on Landscape.